### PR TITLE
donut tooltip

### PR DESCRIPTION
**Pull request by SharonV33**
some minor changes + redo the donut tool tip based on new design
image

### CODE REVIEW

**Review by johancvb — APPROVED** [review]
👏🏼

File: src/components/molecules/Comparer.svelte [hunk 1]
```diff
@@ -12,7 +12,7 @@
       allReports = value
     })
 
-  let noto = compareReports(allReports)
+  let data = compareReports(allReports)
```
Comment (veerleprins) [inline src/components/molecules/Comparer.svelte:15]: Nice that you changed this variable name 👍🏼

File: src/components/organisms/DonutSection.svelte [hunk 2]
```diff
@@ -33,15 +27,15 @@
...
+        flex-direction: column;
         justify-content: space-between;
       }
     }
   }
 </style>
 
 <section>
-  <Title isSubtitle>Soort meldingen</Title>
+  <p>Soort meldingen</p>
```
Comment (veerleprins) [inline src/components/organisms/DonutSection.svelte]: What was the reason for changing this subtitle to a p tag?

File: src/components/atoms/DonutChart.svelte [hunk 3]
```diff
@@ -142,16 +142,25 @@
...
+      padding: 0.5em;
+      width: 100%;
     }
   }
 
+  @include size-m {
+    #tooltip {
+
+    }
+  }
```
Comment (veerleprins) [inline src/components/atoms/DonutChart.svelte]: If there is nothing in here, can't it be removed? Or do you need it for later?

**Review by veerleprins — COMMENTED** [review]
Looking good! 👍🏼  Maybe you can only handle the small title in the tooltip (that of 'Aantal meldingen')

**Review by veerleprins — APPROVED** [review]
Thanks for fixing these small things! Looking good 👍🏼

**Pull request merged**